plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/security/NugetApiKeyComponent.groovy

```diff
@@ -10,7 +10,9 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package com.sonatype.nexus.repository.nuget.internal.security
+package com.sonatype.nexus.repository.nuget.security
+
+import com.sonatype.nexus.repository.nuget.security.NugetApiKeyStore
```

Comment:
There are several dangling imports from the refactoring that should be cleaned up ^